CHANGELOG

```diff
@@ -1,3 +1,10 @@
+1.0.0
+-----
+
+* Updated MicroPython firmware
```

Comment:
Based on the PR diff, it looks like the MicroPython firmware is the same between bbcmicrobit/PythonEditor@master and python-editor@1.0.0.

Reply:
Hmm.. I'd based that from the commit message in https://github.com/bbcmicrobit/PythonEditor/pull/84/commits/c4ef9c224ccd0346fe139e0a35d422eedc15c105

Reply:
Is the hex file from commit https://github.com/bbcmicrobit/PythonEditor/commit/c4ef9c224ccd0346fe139e0a35d422eedc15c105 the same as in https://github.com/bbcmicrobit/PythonEditor/blob/master/firmware.hex ?

Reply:
Yes, firmware I've got in my clean branch is from master not the commit
I think I must have checked out the wrong file

Reply:
Okay, so I am looking at the v1.0.0 release: https://github.com/microbit-foundation/python-editor/tree/1.0.0

And the hex file doesn't look like it is the same than this PR:

https://github.com/microbit-foundation/python-editor/blob/1.0.0/firmware.hex
https://github.com/bbcmicrobit/PythonEditor/blob/23d73277853c621ada1343cd4fca1e5ff0d026e9/firmware.hex

Is it possible there is a missing commit to cherry pick on top of c4ef9c2?

Reply:
I've managed to mess it up in the final commit that should have just updated the CHANGELOG and README, will amend and push!

Reply:
Right, so then the v1.0.0 hex file is the same as bbcmicrobit/PythonEditor@master, correct?
https://github.com/microbit-foundation/python-editor/blob/1.0.0/firmware.hex
https://github.com/bbcmicrobit/PythonEditor/blob/master/firmware.hex

Reply:
Yeah I think this is right now, I've updated the CHANGELOG
view-source:https://python-editor-1-0-0.microbit.org/

Reply:
Great, thank you Sam!